src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/bitbucket/client/model/BitbucketConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package com.github.mc1arke.sonarqube.plugin.ce.pullrequest.bitbucket.client.model;
+
+public class BitbucketConfiguration {
+    private final String url;
+    private final String token;
+    private final String repository;
+    private final String project;
+
+    public BitbucketConfiguration(String url, String token, String repository, String project) {
+        this.url = url;
+        this.token = token;
+        this.repository = repository;
+        this.project = project;
+    }
+
+    public String getRepository() {
+        return repository;
+    }
+
+    public String getToken() {
+        return token;
+    }
+
+    public String getUrl() {
+        return url;
+    }
+
+    public String getProject() {
+        return project;
+    }
+
+    public boolean isCloud() {
+        return url.contains("https://api.bitbucket.org");
```

Comment:
Should this be `startsWith` rather than `contains`? Given URLs are also case insensitive, I'd also suggest adding a `toLowerCase(Locale.ENGLISH)` in there.